rewrite-java/src/main/java/org/openrewrite/java/ReorderMethodArguments.java

```diff
@@ -144,12 +146,19 @@ public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, Execu
                         // this is a varargs argument
                         List<JRightPadded<Expression>> varargs = originalArgs.subList(fromPos, originalArgs.size());
                         reordered.addAll(varargs);
+                        for (int j = 0; j < varargs.size(); j++) {
+                            reorderedNames.add(name + j);
```

Comment:
Not sure what to do here, since this is the varargs argument, and can have N arguments, I've decided to put in the names the name of the original vararg + index.